### ISSUE TITLE
aria-hidden="false" added to all nodes

### ISSUE THREAD

**Issue by joe-watkins** [ableplayer/ableplayer#252]
Line #6111 adds `aria-hidden` to all dom nodes.  Using the inspector visit your demo page and look at the content above the player. http://ableplayer.github.io/ableplayer/demos/video2.html

```
$('body > *').not('.able-modal-overlay').not('.able-modal-dialog').attr('aria-hidden', 'false');
```

Not sure what you were trying to do here.

**Comment by joe-watkins** [comment]
I believe this creates some bugs in Safari 10 which makes controls not focusable by keyboard.

**Comment by terrill** [comment]
@joe-watkins the purpose of that code is to make modal dialogs (i.e., the Preferences dialogs) modal to screen reader users. When a Preferences dialog has focus, it has aria-hidden="false", and everything else in the DOM has aria-hidden="true". That's a standard method for making modal dialogs accessible. Without this screen readers can access the content in the background. 

It shouldn't have any effect on keyboard focus. All ARIA is supposed to do, including the aria-hidden attribute,  is communicate roles, states, and properties to accessibility APIs. That should have no effect on people who aren't using assistive technology. 

What are you observing exactly in Safari 10?  If there are elements that should be focusable but aren't, it would be great if you could create a new issue with the details. I haven't upgraded to Safari 10 yet, but I'm not observing any problems in Safari 9 (OS X 10.11.2, El Capitan) with the demo you referenced.

**Comment by joe-watkins** [comment]
@terrill Tnx for the info -- good call to help with keyboard trap. Could this happen only when needed e.g. user triggers modal `aria-hidden` is then added to nodes and modal is visible and then removed when modal is closed?

Separate issue coming for Safari.. you are correct it does feel like a separate bug :)

Tnx for a great player.

**Comment by terrill** [comment]
@joe-watkins yes, I agree it would be better/leaner to add/remove the attribute. I don't recall why I didn't do that to begin with - I suspect there might have been a reason after testing with various assistive technologies. I'll try again though and change this if it works.

**Comment by terrill** [comment]
Fixed in af562d128087b3aa6e24a7255b1815b645a6fc08

**Comment by mably** [comment]
This breaks other accessible modals that could be present on page.

They will now be displayed as soon as you have an AblePlayer on the page as their `aria-hidden=true` attribute will be removed.

How can we fix this?
